birdspotter/birdspotter/dataio/scripts/get_user_datasets.py

```diff
@@ -1,5 +1,33 @@
 from birdspotter.dataio.models import Dataset
-
+from birdspotter.dataio.models import Shapefile
+from birdspotter.accounts.models import User
 
 def get_datasets_for_user(user):
-	return Dataset.objects.filter(owner_id=user.id).values()
+    return Dataset.objects.filter(owner_id=user.id).values()
+
+def get_dataset_data(user):
+    print("ISADMIN:",user)
+    shapefile_lines = Shapefile.objects.filter(data_set=1)
```

Comment:
Hardcoding the dataset wasn't mentioned as a known issue w/ this PR, so I'm going to open an issue about this to make sure it doesn't get lost in the shuffle. I'd suggest setting a parameterized URL to pass the dataset UUID through the URL (take a look at the queueing branch, or at master once I've got that completed/merged)